.vscode/settings.json

```diff
@@ -1,3 +1,4 @@
 {
-  "typescript.tsserver.experimental.enableProjectDiagnostics": true
+  "typescript.tsserver.experimental.enableProjectDiagnostics": true,
+  "sarif-viewer.connectToGithubCodeScanning": "on"
```

Comment:
This is an unrelated change and should probably be reverted.

Reply:
Oops. Yes. Unfortunately, this file is versioned and the tooling I use to produce these changes involves that flag.